cli/cmd/testdata/install_output.golden

```diff
@@ -135,12 +132,11 @@ spec:
     metadata:
       annotations:
         CreatedByAnnotation: CliVersion
-        conduit.io/created-by: conduit/cli latest
-        conduit.io/proxy-version: latest
+        conduit.io/created-by: conduit/cli undefined
```

Comment:
not specific to this change, we should rethink our use of `CreatedByAnnotation`. it's being overloaded for both `conduit install` and `conduit inject`.

it's possible a user could run a `conduit install` with one version of conduit, and then `conduit inject` with another version. we should be able to identify this by looking at someone's yaml. as the code is written today, the `conduit inject` command would override the `CreatedByAnnotation` from the `conduit install` command.

i've filed #435 to track this.